src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -205,17 +230,18 @@ const EditHomepage = ({ match }) => {
       }
 
       try {
-        const {
+        let {
           content: { frontMatter },
-          sha,
         } = homepageData
+        const { sha } = homepageData
```

Comment:
nit: this change appears to have different semantics; `content` now appears to be writable now whereas previously it was only readable

Reply:
Hey yes this is the case. There was a bug when the key was undefined, but its still valid frontmatter state. As such, I just refine an empty array so that our FE does not throw any error.